.github/workflows/build-nix-on-ubuntu.yml

```
@@ -0,0 +1,27 @@
+name: CMake Build (Nix on Ubuntu x86-64)
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
```

```suggestion
    - uses: actions/checkout@v2
```
If you need all commits:
```suggestion
    - uses: actions/checkout@v2
      with:
        fetch-depth: 0
```